### ISSUE TITLE
`jsx/jsx-props-no-multi-spaces`: not enforce between last prop and self-closing tag

### ISSUE THREAD

**Issue by easternmotors** [eslint-stylistic/eslint-stylistic#301]
### Validations

- [X] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### Describe the bug

When enabling `jsx/jsx-props-no-multi-spaces`:

```
// Desired end state
<Component
    propA="foo"
    propB="bar"
/>

// This properly shown an error - Expected no line gap between “Component” and “propA”
<Component

    propA="foo"
    prop2="bar"
/>

// This properly shown an error - Expected no line gap between “propA” and “propB”
<Component
    propA="foo"

    propB="bar"
/>

// This does not show an error
// I believe would expect the lint error message to say - Expected no line gap between “propB” and self-closing tag
<Component
    propA="foo"
    propB="bar"

/>
```

### Reproduction

The repro link isn't set up for Typescript/JSX

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests

**Comment by 9romise** [comment]
As noted in the [When not to use it](https://eslint.style/rules/jsx-props-no-multi-spaces#when-not-to-use-it), `jsx-props-no-multi-spaces` can be replaced by `no-multi-spaces` which passes all test cases.
